example/go-zero/README.md

```diff
@@ -0,0 +1,17 @@
+# Go-zero api example
+
+Add these below config to `api/etc/api-api.yaml`, to enable uptrace report for go-zero api
+
+```yaml
+
+Telemetry:
+  Name: sdkrpc.rpc
+  Endpoint: uptrace.apm:14317
+  Sampler: 1.0
+  Batcher: otlpgrpc
+  OtlpHeaders:
+    uptrace-dsn: http://project2_secret_token@uptrace.apm:14317/2
+
+```
+
+> https://uptrace.dev/get/ingest/opentelemetry.html#already-using-opentelemetry-sdk
```

Comment:
Thanks! I will also add an entry about go-zero to https://uptrace.dev/opentelemetry/instrumentations/

But could we make this example runnable? E.g. we could change this example to use Uptrace from [Docker](https://github.com/uptrace/uptrace/tree/master/example/docker) so users can run this with few commands.

We also need some way to trigger a request, for example, using a browser (open some URL) or by providing a `client.go` that makes some requests.

What do you think?

Reply:
Done~

Reply:
![image](https://user-images.githubusercontent.com/5897356/237016962-bc441e95-fb4b-4312-9a93-8eb0dc55e140.png)